hazelcast/src/main/java/com/hazelcast/config/EvictionConfig.java

```diff
@@ -58,6 +66,34 @@
      */
     public static final EvictionPolicy DEFAULT_EVICTION_POLICY = EvictionPolicy.LRU;
 
+    /**
+     * Maximum Size Policy
```

Comment:
This documentation doesn't provide anything what the class name already provided :) So can you add some extra documentation behind the purpose of this policy?